scripts/docs/snippetadapter.js

```diff
@@ -117,7 +118,8 @@ function getWebpackConfig( config ) {
 			new webpack.BannerPlugin( {
 				banner: bundler.getLicenseBanner(),
 				raw: true
-			} )
+			} ),
+			new webpack.DefinePlugin( config.definitions )
```

Comment:
Maybe we should do `JSON.stringify` for each value automatically?

Reply:
Hm. We can do it.

Reply:
Actually, It's weird for me that this plugin doesn't do it out of the box.